src/rmp/test/aes_nangate45.tcl

```diff
@@ -0,0 +1,8 @@
+# For debugging optimizations
```

Comment:
Does this need to be committed?  Likewise the sdc

Reply:
I would find it helpful if they were committed, but I'll remove them if you feel strongly